src/Terminus.php

```diff
@@ -201,6 +203,8 @@ private function addBuiltInCommandsAndHooks()
     private function configureContainer(ContainerInterface $container)
     {
         // Add the services.
+        $container->add(Client::class);
+        $container->add(HttpRequest::class);
```

Comment:
Yeah. That thing is getting overloaded. But there's no easy way to test our Request class if it's dependencies aren't injected.